model/etcd.go

```diff
@@ -129,15 +129,15 @@ func (e Etcd) SystemdUnitName() string {
 	return "etcd2.service"
 }
 
-// Version returns the version of etcd (e.g. `2`, `3`, `3.1.3`) to be used for this etcd cluster
+// Version returns the version of etcd (e.g. `2`, `3`, `3.1.4`) to be used for this etcd cluster
 func (e Etcd) Version() EtcdVersion {
+	if e.Cluster.Version == "3" {
+		return "3.1.5"
+	}
 	if e.Cluster.Version != "" {
```

Comment:
The intention here is to translate:
```yaml
etcd:
  version: 3
```
to
```yaml
etcd:
  version: 3.1.5
```
for users who doesn't care much about minor/patch versions.
However as kube-aws starts defaulting to etcd3 from now on, we can just default to "3.1.5" when it is omitted, as you've suggested 👍